README.md

```diff
@@ -144,6 +144,28 @@ http://dockerhost:5000
 When you install docker-osx-dev, it adds an entry to your `/etc/hosts` file so
 that `http://dockerhost` works as a URL for testing your Docker containers.
 
+# docker-machine support
+
+`docker-machine` support is experimental. You can use it as the way it is used for
+`boot2docker`, but run `docker-machine env` before. So as an example, run as:
+
+```
+> docker-machine create --driver virtualbox <machine-name>
+> eval "$(docker-machine env <machine-name>)"
+> docker-osx-dev install
```

Comment:
So, I am little confused on what should be the role of the `install` for `docker-machine`.

```bash
   install_dependencies # Should actually run before docker-machine create. Once for osx user.
   init_docker_host # Per machine, should run after docker-machine create. This seems only important for the next command
   install_rsync_on_docker_host # Is this really important? the sync will also install rsync.
   add_docker_host # Per machine, should run after docker-machine create. Also, is it important?
   add_environment_variables # Not used for docker-machine
```

I would avoid making the `install` create the docker-machine, as in the creation the user needs to supply the driver he wants and all the driver's configuration. It is also seems misleading that a `install` command would create a machine.

I'd suggest the following flow:

```
docker-osx-dev install_dependencies # once per osx user
docker-machine create --driver virtualbox <machine-name>
eval "$(docker-machine env <machine-name>)"
cd /foo/bar
docker-osx-dev
```

In this way there is no need to run `install` after `docker-machine create`. The `install` would only make sense for `boot2docker`.